src/mpyl/projects/versioning.py

```diff
@@ -112,6 +112,18 @@ def upgrade(self, previous_dict: ordereddict) -> ordereddict:
         return previous_dict
 
 
+class ProjectUpgraderOne31(Upgrader):
+    target_version = "1.3.1"
+
+    def upgrade(self, previous_dict: ordereddict) -> ordereddict:
+        if deployment := previous_dict.get("deployment", {}):
```

Comment:
you can safely chain these as previous_dict.get("deployment", {}).get("kubernetes", {}).get("cmd", None)